mosec/worker.py

```diff
@@ -51,35 +62,49 @@ class Worker(abc.ABC):
     will follow the "_same type_" constraint.
     """
 
+    # pylint: disable=no-self-use
+
     example: Any = None
-    _id: int = 0
+    _worker_id: int = 0
 
     def __init__(self):
-        self._stage = None
-        self._max_batch_size = 1
+        self._stage: str = ""
+        self._max_batch_size: int = 1
```

Comment:
May I know why it has both class attributes and instance attributes? @lkevinzc

Reply:
I think it's because `example` and `id` can be shared at the same stage across difference instances? Could you please share your concern on this part?

Reply:
`worker_id` cannot be shared. Different instances will have different worker IDs. Now it works because it's copied to another Process before it's affected by the next worker.

* pros
  * can move all to class attribute to avoid user forgetting the `super().__init__()`
  * strong controlled by the library
* cons
  * a little bit hack